### ISSUE TITLE
Authorization doesn't work

### ISSUE THREAD

**Issue by zvrba** [jenius-apps/nightingale-rest-api-client#164]
Installed Nightingale from app store. Tried to use access token in three ways:

1. Choosing "Bearer" under auth and pasting access token there.
2. Choosing "OAuth2.0" under auth and pasting access token there.
3. Turned off auth and manually set up Authorization header with "Bearer <token>" value.

Nothing worked, I always get "401 unauthorized" back. Copy-pasting the same token into Postman triggers the API call successfully. Though probably not the issue, in the log window I noticed that Nightingale adds a "Cokkie ARRAffinity" header that Postman does not.

**Comment by dpaulino** [comment]
Are you able to share your URL?

**Comment by zvrba** [comment]
Yes, but how do I send a private message on github?

**Comment by dpaulino** [comment]
Feel free to email at jenius_apps@outlook.com

**Comment by zvrba** [comment]
Sent to email.

**Comment by dpaulino** [comment]
Thanks, just so I understand, this is an API that you're building yourself right? Does the error repro in both localhost and with the link you sent?

**Comment by zvrba** [comment]
Yes, it's an API I'm building myself. It's a plain ASPNET Core api configured to verify tokens issued by Azure AD. I do not test against localhost, I use only remote debugging.
